packages/manager/apps/web/client/app/private-database/onboarding/translations/Messages_fr_FR.json

```diff
@@ -2,10 +2,10 @@
   "private_databases_onboarding_title": "Bases de données",
   "private_databases_onboarding_description": "Vous avez la possibilité d’ajouter des bases de données  Start SQL MySQL supplémentaires à votre hébergement. Augmentez la puissance de votre hébergement ainsi que vos possibilités de stockage. Les bases de données MySQL permettent également une gestion optimisée de vos données. Cette solution est idéale pour booster vos hébergements Perso ou Pro.",
   "private_databases_onboarding_description2": "Vous avez également la possibilité d’opter pour des bases de données SQL privées. Vous avez le choix entre nos différentes solutions MySQL, MariaDB ou PostgreSQL. Elles vous apportent davantage de liberté de configuration au sein de votre hébergement.",
-  "private_databases_onboarding_order": "Commander CloudDB",
+  "private_databases_onboarding_order": "Commander une Web Cloud Database",
```

Comment:
Why do we sometimes use the singular or plural when we would say a product name because it is uppercase, so I guess it is invariable.